package.json

```
@@ -2,15 +2,19 @@
 	"name": "absurd-use",
 	"version": "0.0.1",
 	"private": true,
+	"type": "module",
 	"scripts": {
+		"setup": "npm ci --ignore-scripts && tsc && npm run vendor",
 		"build": "tsc",
 		"watch": "tsc --watch",
-		"serve": "npx http-server ./app",
-		"vendor": "npm ci --ignore-scripts && cd build && npm ci --ignore-scripts && npm run vendor",
+		"serve": "npx http-server --gzip ./app",
```

The flag `--gzip` will not work as per documentation
`npx http-server [path] [options]`

```suggestion
		"serve": "npx http-server ./app --gzip ",
```